services/service-core/src/app.module.ts

```diff
@@ -13,10 +13,29 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+
 
 @Module({
-  imports: [],
+
+  imports: [
+    TypeOrmModule.forRootAsync({
+      imports: [ConfigModule.forRoot({
+        isGlobal: true,
+        envFilePath: ".local.env"
+      })],
+      useFactory: (configService: ConfigService) => ({
+        type: 'postgres',
+        host: configService.get('DB_HOST'),
+        port: +configService.get<number>('DB_PORT'),
```

Comment:
Better to use Number(configService.get<number>('DB_PORT')). Use of `+` is technically correct but very obscure and unclear